packages/webdriverio/src/commands/element/isClickable.ts

```diff
@@ -46,7 +46,11 @@ export async function isClickable (this: WebdriverIO.Element) {
         return false
     }
 
-    if (this.isMobile && await this.getContext() === 'NATIVE_APP') {
+    /**
+     * some Appium platforms don't support the `getContext` method, in that case
+     * we can't determine if we are in a native context or not, so we return undefined
+     */
+    if (this.isMobile && await this.getContext().catch(() => undefined) === 'NATIVE_APP') {
```

Comment:
@christian-bromann 

Why are we handling this in the method? Why not make something re-usable for WDIO so we have the context and we can access it in all methods that are not usable for mobile

- [set it for 95% of the cases](https://github.com/webdriverio/visual-testing/blob/main/packages/service/src/service.ts#L50)
- [the method](https://github.com/webdriverio/visual-testing/blob/main/packages/service/src/utils.ts#L228)
- [command check](https://github.com/webdriverio/visual-testing/blob/main/packages/service/src/service.ts#L70)

Reply:
Mind raising an issue for this? This makes totally sense to extend the environment flag with something like `isNative`.

Reply:
Done, see https://github.com/webdriverio/webdriverio/issues/12208

Reply:
Thanks!